src/__test__/components/data-exploration/hierarchical-tree/HierarchicalTree.test.jsx

```diff
@@ -71,33 +58,41 @@ describe('HierarchicalTree', () => {
     const dropInfo = {
       dragNode: {
         ...firstChild,
+        pos: '0-0-0',
         props: { eventKey: firstChild.key },
       },
       dragNodesKeys: [firstChild.key],
       dropPosition: 2,
       dropToGap: true,
       node: {
         ...thirdChild,
+        pos: '0-0-3',
         props: { eventKey: thirdChild.key },
       },
     };
 
-    const mockOnHierarchyUpdate = jest.fn();
+    const mockOnCellSetReorder = jest.fn();
     const component = mount(
-      <HierarchicalTree experimentId={experimentId} treeData={treeData} onHierarchyUpdate={mockOnHierarchyUpdate} />,
+      <HierarchicalTree
+        experimentId={experimentId}
+        treeData={treeData}
+        onCellSetReorder={mockOnCellSetReorder}
+      />,
     );
     waitForComponentToPaint(component);
     const tree = component.find('HierarchicalTree Tree');
+
     tree.getElement().props.onDrop(dropInfo);
     component.update();
 
-    expect(mockOnHierarchyUpdate).toHaveBeenCalledTimes(1);
-    const firstArgument = mockOnHierarchyUpdate.mock.calls[0][0];
-    const childrenKeys = firstArgument[0].children.map((child) => child.key);
-    expect(childrenKeys).toEqual(['2a', '3a', '1a']);
+    expect(mockOnCellSetReorder).toHaveBeenCalledTimes(1);
+    const [cellSetKey, newPosition] = mockOnCellSetReorder.mock.calls[0];
+
+    expect(cellSetKey).toEqual('1a');
+    expect(newPosition).toEqual(1);
   });
 
-  it('Can have child component change parent', () => {
+  it('Can\'t have child component change parent', () => {
```

Comment:
Why do you have to use `\'`, can't you just use "... can't..."?

Reply:
because of the single quotes of the string, and using double quotes would go against what we usually use, right?

Reply:
oh ok